app/models/user.rb

```diff
@@ -140,7 +140,7 @@ def self.search_states
 
   scope :latest, -> {limit(5).order(created_at: :desc)}
 
-  scope :but_impersonation_admin, -> { where(['username <> ?', ThreeScale.config.impersonation_admin['username']]) }
+  scope :but_impersonation_admin, -> { where(['users.username <> ?', ThreeScale.config.impersonation_admin['username']]) }
```

Comment:
why is this needed?

Reply:
I found some tests failing because username was an ambiguous column name: https://app.circleci.com/pipelines/github/3scale/porta/21537/workflows/052cd6c3-150c-44f3-a943-a83579956da4/jobs/249337

Reply:
```suggestion
  scope :but_impersonation_admin, -> { where.has { username <> ThreeScale.config.impersonation_admin['username'] }}
```
What I'm concerned with is hardcoding table name might be a problem with more complicated joins or sub-queries. I'm not sure that's a real risk but this is my reasoning :man_shrugging:

Reply:
done